core/src/zeit/content/image/imagegroup.py

```diff
@@ -167,6 +167,24 @@ def variant_url(self, name, width=None, height=None,
             url += u'__{fill}'.format(fill=fill_color)
         return url
 
+    def from_image(where, name, image):
```

Comment:
Ich kann mir kaum vorstellen, dass das in der Form funktioniert :), ich denke das muss so heißen:

```suggestion
    @classmethod
    def from_image(cls, where, name, image):
```